Sources/MongoKitten/Database.swift

```diff
@@ -447,7 +448,7 @@ extension Database {
         // Check for success
         guard successDocument["ok"] as Int? == 1 else {
             logger.error("Authentication for MongoDB user \(details.username) with MongoCR failed against \(details.database) for the following reason")
-            logger.error(document)
+            logger.error((successDocument["errmsg"] as String?)!)
```

Comment:
I think it would be good to have a human-readable error message by default. For debugging purposes one could inspect the underlying document off-course. I've reverted the change, to keep error handling in line with the rest of the codebase.